### ISSUE TITLE
Slider - tokenise component and finalise design

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#268]
#### Description / User Story

We currently have the design and the code for the slider component (single & multi value), but not all tokens are actually used in the component. Also the design page is not yet finalised.

With #437 we also have a new task to re-define the scope of the component and possibly to combine all current variants into one single slider component. Once this task is done, we probably also need to change the existing design and code again.

After #461 we also can implement the tooltip component into the slider, resulting in yet another possible design/code changes.

----
#### Requirements
- [ ] #437 
- [x] #461 

----
#### Acceptance Criteria
- [ ] The defined scope from #437 is implemented in the designed component(s)
- [ ] The defined scope from #437 is implemented in the coded component(s)
- [ ] The designed component(s) use(s) the tooltip
- [ ] The coded component(s) use(s) the tooltip
- [ ] All design tokens needed for the component(s) are defined in the design
- [ ] All defined tokens are used in the coded components
- [ ] "showLegend" prop has been renamed to "hasLegend" in design and code
- [ ] All required events have been implemented into the component (i.e. onChange, onFocus, onBlur, onClickMinMax)
- [ ] [Props Excel](https://ts.accenture.com/:x:/r/sites/b01ler-core/Shared%20Documents/Documentation_Props.xlsx?d=wb2cc9fb85c3940499652a4e6bbe09ef1&csf=1&web=1&e=bhRYao) has been updated / comments have been resolved and props changed from red to black font color

----
#### Background information
- Initial ticket for slider: #114 
- Required slider tickets before this one can be done: #437
- Other tickets for slider #464, #512, 522, 327
